docs/asciidoc/graph-updates/periodic-iterate.adoc

```diff
@@ -8,32 +8,68 @@ ifdef::backend-html5[]
 endif::[]
 
 The `apoc.periodic.iterate` procedure is helpful when you need to handle large amounts of data for import, refactoring, and other cases that require large transactions.
-It provides a way to batch the data by dividing the workload into two parts - a data-driven statement and an operations statement.
+It provides a way to batch the data by dividing the workload into two parts:
 
-The data-driven statement is how you select what data needs handled.
+a data-driven statement:: This defines how you select what data needs handled.
 You can provide a Cypher statement to select from existing graph data, read external data from a file or API, or retrieve data from another datastore.
-The operations statement is what you want done to the selected data.
+
+an operation statement:: This defines what you want done to the selected data.
 You can do things like execute Cypher for updating or creating/deleting the data or run other procedures to manipulate and transform values before loading.
 
-To run `apoc.periodic.iterate`, the data-driven statement is provided as the *first*, outer statement that results in a stream of values to be processed.
-The operations statement is provided as the *second*, inner statement to process *one* element at a time or (with `iterateList:true`) a batch at a time.
+The data-driven statement is provided as the *first*, outer statement that results in a stream of values to be processed.
+The operations statement is provided as the *second*, inner statement to process *one* element at a time or (with `batchMode: "BATCH"`) a batch at a time.
 The results of the outer statement are passed to the inner statement as parameters, so they are automatically made available with their names.
 
-.configuration options
+.Config
+[options=header]
+|===
+| name | type | default | description
+| batchSize | Long | 10000 | run the specified number of operations statements in a single tx - params: {_count, _batch}
+| parallel | boolean | false | run operations statements in parallel (note that statements might deadlock if conflicting)
+| retries | Long | 0 | if the operations statement fails with an error, sleep 100ms and retry until retries-count is reached - param {_retry}
+| batchMode | String | "BATCH" a| how data-driven statements should be processed by outer statement. Valid values are:
```

Comment:
Should it be how the data-driven statement is processed by the INNER statement? Because the data-driven statement = outer statement?

Reply:
Haha, I was getting so confused by outer and inner! I've removed those words and replaced with data-driven and operation now